src/main/asciidoc/batch/file-ingest-sftp/kubernetes.adoc

```diff
@@ -0,0 +1,278 @@
+On the SFTP server create a directory called `remote-files`, for example:
++
+```
+sftp> mkdir remote-files
+```
++
+. Create a shared NFS directory
++
+Create a read/write directory on the NFS server.
++
+```
+$ sudo mkdir /export/shared-files
+$ sudo chmod 0777 /export/shared-files
+```
+. Register the `sftp-dataflow` source and the `tasklauncher-dataflow` sink
++
+With our Spring Cloud Data Flow server running, we register the `sftp-dataflow` source and `task-launcher-dataflow` sink.
+The `sftp-dataflow` source application will do the work of polling the remote directory for new files and downloading them to the local directory.
+As each file is received, it emits a message for the `task-launcher-dataflow` sink to launch the task to process the data from that file.
++
+In the Spring Cloud Data Flow shell:
++
+[source,console,options=nowrap]
+----
+dataflow:>dataflow:>app register --name sftp --type source --uri docker:springcloud/sftp-dataflow-source-kafka --metadata-uri maven://org.springframework.cloud.stream.app:sftp-dataflow-source-kafka:jar:metadata:2.1.0.BUILD-SNAPSHOT
+Successfully registered application 'source:sftp'
+dataflow:>app register --name task-launcher --type sink --uri docker:springcloud/task-launcher-dataflow-sink-kafka --metadata-uri maven://org.springframework.cloud.stream.app:task-launcher-dataflow-sink-kafka:jar:metadata:1.0.0.BUILD-SNAPSHOT
+Successfully registered application 'sink:task-launcher'
+----
++
+. Register and create the file ingest task:
+[source,console,options=nowrap]
+dataflow:>app register --name fileIngest --type task --uri docker:springcloud/ingest
+Successfully registered application 'task:fileIngest'
+dataflow:>task create fileIngestTask --definition fileIngest
+Created new task 'fileIngestTask'
++
+. Create and deploy the stream
++
+Now lets create and deploy the stream.
+Once deployed, the stream will start polling the SFTP server and, when new files arrive, launch the batch job.
++
+NOTE: Replace `<user>`, '<pass>`, and `<host>` below.
+The `<host>` is the SFTP server host, `<user>` and `<password>` values are the credentials for the remote user. 
+Additionally, replace `--spring.cloud.dataflow.client.server-uri=http://<dataflow-server-ip>` with the Cluster IP (External IP should work as well) of your dataflow server, as shown by `kubectl get svc/scdf-server`. 
+The default Data Flow server credentials are `user` and `password`.
++
+Notice how we use the Kubernetes Persistent Volume Claim(PVC) resource that we created earlier. 
+In the stream definition, the PVC and the associated Volume Mount are passed to the task via `--task.launch.request.deployment-properties`. 
+The `deployer.\*.kubernetes...` properties provide native Kubernetes specs as JSON to instruct the Data Flow server's deployer to add this configuration to the container configuration for the pod that will run the batch job. 
```

Comment:
there is a`\` in `deployer.\*.kubernetes..`

Reply:
Fixed